src/EventStore.Core.Tests/Helpers/MiniClusterNode.cs

```diff
@@ -112,21 +112,48 @@ public class MiniClusterNode<TLogFormat, TStreamId> {
 					ReadOnlyReplica = readOnlyReplica
 				},
 				Interface = new() {
+#pragma warning disable 618
```

Comment:
You should be able to just set the new options here rather than setting both the new and deprecated ones

Reply:
Done